azurelinuxagent/common/protocol/wire.py

```diff
@@ -488,7 +488,7 @@ def put_page_blob(self, url, data):
 
 
 def event_param_to_v1(param):
-    param_format = '<Param Name="{0}" Value={1} T="{2}" />'
+    param_format = ustr('<Param Name="{0}" Value={1} T="{2}" />')
```

Comment:
can we add utf_8 somewhere in the function name to be more explicit on what it returns ?

Reply:
Done